Purchase/Creatable.ts

```diff
@@ -15,4 +16,7 @@ export namespace Creatable {
 			typeof value.buyer == "string"
 		)
 	}
+	export function validate(value: Creatable) {
```

Comment:
This doesnt need limit to pass down to amount?

Reply:
indeed it does.